Package.swift

```diff
@@ -68,6 +72,11 @@ let stubDataBuildSettings: [CXXSetting] = buildSettings.appending([
     .headerSearchPath("."),
 ])
 
+let linkerSettings: [LinkerSetting] = [
+    .linkedLibrary("wasi-emulated-signal"),
+    .linkedLibrary("wasi-emulated-mman"),
```

Comment:
Can we make these conditional on WASI too, as the other settings?

Reply:
Oops, nice catch